deployment/CP05MOAS-GL559_Deploy.csv

```diff
@@ -4,4 +4,10 @@ TS20210131,,TS20210419,,CP05MOAS-GL559-01-ADCPAM000,1,1,2021-01-31T18:01:00,2021
 TS20210131,,TS20210419,,CP05MOAS-GL559-02-FLORTM000,1,1,2021-01-31T18:01:00,2021-04-19T17:44:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-FLORTM-03921,39.92493,-71.1781,,1000,1000,FZ-1,,
 TS20210131,,TS20210419,,CP05MOAS-GL559-03-CTDGVM000,1,1,2021-01-31T18:01:00,2021-04-19T17:44:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-CTDGVM-09029,39.92493,-71.1781,,1000,1000,FZ-1,,
 TS20210131,,TS20210419,,CP05MOAS-GL559-04-DOSTAM000,1,1,2021-01-31T18:01:00,2021-04-19T17:44:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-DOSTAM-00436,39.92493,-71.1781,,1000,1000,FZ-1,,
-TS20210131,,TS20210419,,CP05MOAS-GL559-05-PARADM000,1,1,2021-01-31T18:01:00,2021-04-19T17:44:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-PARADM-50145,39.92493,-71.1781,,1000,1000,FZ-1,,
\ No newline at end of file
+TS20210131,,TS20210419,,CP05MOAS-GL559-05-PARADM000,1,1,2021-01-31T18:01:00,2021-04-19T17:44:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-PARADM-50145,39.92493,-71.1781,,1000,1000,FZ-1,,
+TS20210517,,,,CP05MOAS-GL559-00-ENG000000,2,1,2021-05-17T20:34:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGCON-GLDRCP-00559,39.81802,-70.38575,,1000,1000,SS-1,,
+TS20210517,,,,CP05MOAS-GL559-01-ADCPAM000,2,1,2021-05-17T20:34:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-ADCPAM-43106,39.81802,-70.38575,,1000,1000,SS-1,,
+TS20210517,,,,CP05MOAS-GL559-02-FLORTM000,2,1,2021-05-17T20:34:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-FLORTM-03921,39.81802,-70.38575,,1000,1000,SS-1,,
+TS20210517,,,,CP05MOAS-GL559-03-CTDGVM000,2,1,2021-05-17T20:34:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-CTDGVM-09029,39.81802,-70.38575,,1000,1000,SS-1,,
+TS20210517,,,,CP05MOAS-GL559-04-DOSTAM000,2,1,2021-05-17T20:34:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-DOSTAM-00364,39.81802,-70.38575,,1000,1000,SS-1,,
+TS20210517,,,,CP05MOAS-GL559-05-PARADM000,2,1,2021-05-17T20:34:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-PARADM-50145,39.81802,-70.38575,,1000,1000,SS-1,,
```

Comment:
QLR deploy table lists SS-2 for assigned line (but the main part of the QLR lists SS-1).  Need to verify which is correct. The rest looks good.

Reply:
559 is on the SS-1 line. Deploy CSV correct. fixed QLR.

Reply:
roger that!  looks good then.

Reply:
Looks good.